System/OsPath/Data/ByteString/Short.hs

```diff
@@ -173,3 +175,17 @@ module System.OsPath.Data.ByteString.Short (
   ) where
 
 import Data.ByteString.Short.Internal
+import System.OsPath.Data.ByteString.Short.Internal
+
+import Prelude (Maybe(..), Ord(..), Num(..), ($), otherwise)
+import Data.Word (Word8)
+
+uncons2 :: ShortByteString -> Maybe (Word8, Word8, ShortByteString)
```

Comment:
This seems obsolete with [unconsN](https://github.com/haskell/bytestring/pull/525) no?

Reply:
And that PR also suggests that the combination of split/unpack is faster.

Reply:
Specialised function without loops or lists is faster than both `unconsN` or `split` / `unpack`. And this one is used in a hot loop.